anvil/anvil.go

```diff
@@ -23,6 +23,8 @@ type Config struct {
 }
 
 type Anvil struct {
+	RpcClient *rpc.Client
```

Comment:
nit: I don't see this field being used outside of this package, should it be private?